common/app/views/fragments/atoms/media.scala.html

```diff
@@ -0,0 +1,24 @@
+@import conf.Configuration.site.host
+@import views.support.RenderClasses
+
+@(media: model.content.MediaAtom)(implicit request: RequestHeader)
+
+@defining(media.assets.head.platform == "Youtube") { isYouTube =>
```

Comment:
Can we differentiate YoutubeAtom and MediaAtom and extract this logic out of this template? For instance something like this in `atom.scala.html`
```
model match {
    case quiz: Quiz => views.html.fragments.atoms.quiz(quiz, maybeResults = None, false, Nil)
    case media: YoutubeAtom => views.html.fragments.atoms.youtube(media, origin)
    case media: MediaAtom => views.html.fragments.atoms.media(media)
    case _ =>		        
```